### PR TITLE
adding back in the generate jwk using api step

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** adding back in the generate jwk using api step
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-489431](https://oktainc.atlassian.net/browse/OKTA-489431)

Netlify Link:
https://62588c64c3fcad1e44420da6--reverent-murdock-829d24.netlify.app/docs/guides/implement-oauth-for-okta-serviceapp/main/

### CODE REVIEW

**Review by venkatviswanathan-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md [hunk 1]
```diff
@@ -41,44 +41,55 @@ The following are the high-level steps required to perform the Client Credential
...
 
-    > **Note:** You can also use the `/oauth2/v1/clients` endpoint to [create your service app](/docs/reference/api/oauth-clients/#request-example-create-a-service-app-with-a-jwks). If you use the API, generate your public/private JWKS key pair first.
-
 4. Enter a name for your app integration and click **Save**.
 
-## Generate the JWK using the Okta apps API
+## Generate the JWK using the API
 
-The Okta apps API provides functionality to manage JWKs, including generating new ones:
+The `private_key_jwt` client authentication method is the only supported method for OAuth service apps that want to get access tokens with Okta scopes.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md:60]:
```suggestion
The `private_key_jwt` client authentication method is the only supported method for OAuth 2.0 service apps that want to get access tokens with Okta scopes.
```

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md [hunk 2]
```diff
@@ -41,44 +41,55 @@ The following are the high-level steps required to perform the Client Credential
...
 
-<ApiOperation method="post" url="/api/v1/apps/${applicationId}/credentials/jwks" />
+The private key that you use to sign the JWT must have the corresponding public key registered in the [JWKSet](/docs/reference/api/oauth-clients/#json-web-key-set) of the OAuth service app. We recommend generating the public/private key pair first before creating the OAuth service app.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md:62]:
```suggestion
The private key that you use to sign the JWT must have the corresponding public key registered in the [JWKSet](/docs/reference/api/oauth-clients/#json-web-key-set) of the OAuth 2.0 service app. Generate the public/private key pair first before creating the OAuth 2.0 service app.
```

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md [hunk 3]
```diff
@@ -41,44 +41,55 @@ The following are the high-level steps required to perform the Client Credential
...
+    * Key use: signature
+    * Algorithm: RSA256
+    * Key ID: This can be any random value.
+
+> **Note:** Use the JSON Web Key Generator link to generate a JWKS public/private key pair for testing purposes only. For a production use case, use your own internal instance of the key pair generator. See this [key pair generator](https://github.com/mitreid-connect/mkjwk.org) for an example.
 
-A successful request returns a generated JWK. See [Add new JSON Web Key](/docs/reference/api/apps/#add-new-json-web-key).
+2. The JSON Web Key Generator tool extracts the public key from the key pair automatically. For testing purposes, copy the Public Key that is provided.
+
+> **Note:** Some Okta SDKs require that keys be in Privacy Enhanced Mail (PEM) format. If you are working with an Okta SDK that requires that the key be in PEM format, after you have generated the key pair, copy the public/private key pair into a [JWK to PEM Convertor tool](https://8gwifi.org/jwkconvertfunctions.jsp) and copy the private key to use when signing the JWT.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md:75]:
```suggestion
> **Note:** Some Okta SDKs require that keys be in Privacy Enhanced Mail (PEM) format. If you are working with an Okta SDK that requires that the key be in PEM format, after you have generated the key pair, copy the public/private key pair into a [JWK to PEM Convertor tool](https://8gwifi.org/jwkconvertfunctions.jsp) and use the PEM formatted private key when signing the JWT.
```

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md [hunk 4]
```diff
@@ -41,44 +41,55 @@ The following are the high-level steps required to perform the Client Credential
...
 2. In the Admin Console, go to **Applications** > **Applications**, and then click **Create App Integration**.
 
 3. On the Create a new app integration page, select **API Services** as the **Sign-in method** and click **Next**.
 
-    > **Note:** You can also use the `/oauth2/v1/clients` endpoint to [create your service app](/docs/reference/api/oauth-clients/#request-example-create-a-service-app-with-a-jwks). If you use the API, generate your public/private JWKS key pair first.
-
 4. Enter a name for your app integration and click **Save**.
 
-## Generate the JWK using the Okta apps API
+## Generate the JWK using the API
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md:58]: This is a question for my own knowledge: when do we use JWK vs JWKS?

**Review by vanngo-okta — APPROVED** [review]
Just a few small suggestions. Everything else looks good! Approving 🚀

**Pull request merged**